### ISSUE TITLE
Unable to stream to Hololens from either Spinningcube or Multithreaded servers

### ISSUE THREAD

**Issue by mike-holo** [3DStreamingToolkit/3DStreamingToolkit#283]
## Report
- [x] I have searched [existing issues](https://github.com/3DStreamingToolkit/3DStreamingToolkit/issues) and this is not a duplicate.

## Issues: Hololens scenarios where it fails to stream
Behavior below is identical whether the Spinning cube or Multithreaded servers are being used. I also have complied on two different computers, and see this same result.
* Using Unity 2017.4.4f1
    * I get an error displayed about "unable to load ADVAPI32.DLL". This seems to be a known issue for Unity so I am surprised this version works for other people. See https://issuetracker.unity3d.com/issues/wsa-unable-to-load-dll-advapi32-dot-dll-when-running-an-app-at-runtime-after-httpwebrequest-under-il2cpp
* Using Unity 2018.2 Beta
    * I get connect messages and can see the Hololens client in my Server's list. When clicking to start the Hololens gives connect messages but never shows any stream. The connect messages are laggy as if there is some processing going on. 
* Using Directx hololens client
    * When launched it simply does nothing, no messages and the server does not see a connection. After a long period of time it exits.

## Expected result: To stream as the desktop client does currently
Running the desktop client works for me with the following config
* Using the same signal server as above (the lightweight signal server provided that I hosted at an http address)
* Default resolutions but 30 fps

And it works in multiple scenerios
* Works with Spinning cube or Multithreaded servers
* Works all on local machine or with client / server on two different computers on the same network

Note: though the client works perfectly well it does seem to sometimes disconnect unexpectly, often about 30 secs into the run but this is not consistent 

Thank you for your help!

**Comment by tylergibson** [comment]
I don't believe we support IL2CPP for the Unity client, only the .NET scripting backend.

Also, be sure that your turn server credentials are defined in directx and Unity clients (both currently require hard coded credentials followed by a build and deploy to device)

**Comment by anderm** [comment]
We updated our getting started documentation. Please follow the steps here and you should see the stream on HoloLens: https://3dstreamingtoolkit.github.io/docs-3dstk/#getting-started
